man/changeFileMetaData.Rd

```diff
@@ -0,0 +1,48 @@
+%
+%  Auto-generated file, do not modify.
+%  Instead, copy this file to the man/ folder, remove this warning, and edit freely.
+%  Use Git to identify changes in this file which suggest where to change your edited copy.
+%
+\name{changeFileMetaData}
+\alias{changeFileMetaData}
+\docType{methods}
+\title{
+changeFileMetaData
+}
+\description{
+
```

Comment:
description is missing here as well as in the Python utils.  Add it there to inherit it here.